src/pages/patientView/clinicalInformation/ClinicalInformationMutationalSignatureTable.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from "react";
+import {ClinicalAttribute, ClinicalData} from "../../../shared/api/generated/CBioPortalAPI";
+import LazyMobXTable, {Column} from "shared/components/lazyMobXTable/LazyMobXTable";
+import {fetchMutationalSignatureData} from "../../../shared/lib/StoreUtils";
+import styles from './style/patientTable.module.scss';
+import {SHOW_ALL_PAGE_SIZE} from "../../../shared/components/paginationControls/PaginationControls";
+import {IMutationalSignature} from "../../../shared/model/MutationalSignature";
+import convertSamplesData, {IConvertedSamplesData} from "./lib/convertSamplesData";
+import {ISampleRow} from "./ClinicalInformationSamplesTable";
+import _ from 'lodash';
+
+export interface IClinicalInformationMutationalSignatureTableProps {
+    data: IMutationalSignature;
+    showTitleBar?: boolean;
+    cssClass?:string;
+    showFilter?:boolean;
+    showCopyDownload?:boolean;
+}
+
+class MutationalSignatureTable extends LazyMobXTable<IMutationalSignatureRow> {}
+
+interface IMutationalSignatureRow {
+    mutationalSignatureId:string;
+    [key:string]:string;
+};
+
+export default class ClinicalInformationMutationalSignatureTable extends React.Component<{}> {
+    public render() {
+        const sampleData: IMutationalSignature[] = fetchMutationalSignatureData();
+        const sampleInvertedData =  this.invertData(sampleData);
```

Comment:
I don't see a reason to get sampleInvertedData if you just want to get the columns info which is gonna just be  _.map(_.uniq(_.map(mut, 'sampleId')), sampleId=>({id:sampleId}))